tools/fix-spec

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+# patch given spec file to have the correct version and declare bundled NPM dependencies
+# Usage: fix-spec <spec-file> <version>
+set -eu
+
+spec="$1"
+version="$2"
+
+PROVIDES=$(npm ls --omit dev --package-lock-only --depth 10 |
```

Comment:
I locally updated this to `--depth=Infinity` as in https://github.com/cockpit-project/starter-kit/pull/634 (thanks @marusak for pointing out). I'll push this after getting a review, in case something else needs changing.

Reply:
Pushed.